### PR TITLE
Initial Transmission docs

### PR DESCRIPTION
**Pull request by alamo18**

### CODE REVIEW

File: source/3/3.3/accessories.rst [hunk 1]
```diff
@@ -7,4 +7,5 @@ This chapter contains the list of accessories installed by default.
    :maxdepth: 2
    :titlesonly:
 
+   3.1.4/transmission
```
Comment (tsimonq2) [inline source/3/3.3/accessories.rst]: This shouldn't be here, it should be in the similar file under 3.1.

File: source/3/3.3/3.1.4/transmission.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Chapter 3.3.4: Leafpad
+==============================
+
+Description
+---------------
+Transmission is a BitTorrent client which features a variety of user interfaces on top of a cross-platform back-end.
+
+It is a fully open-source, non-freemium client doesn't bundle toolbars, pop-up ads, flash ads, twitter tools or track users.
```
Comment (tsimonq2) [inline source/3/3.3/3.1.4/transmission.rst]: Non-freemium? O.o

Comment (alamo18) [reply source/3/3.3/3.1.4/transmission.rst]: yeah. https://www.google.com/search?client=ubuntu&channel=fs&q=freemium&ie=utf-8&oe=utf-8

File: source/3/3.3/accessories.rst (hunk 1 above)
Comment (alamo18) [reply source/3/3.3/accessories.rst]: fixing rn

**Comment by tsimonq2** [comment]
I'll fix this up myself and merge. Thanks!

**Comment by alamo18** [comment]
thanks simon

**Pull request closed without merging**